src/CRUD.js

```diff
@@ -67,7 +68,11 @@ exports.CRUD = class CRUD {
           case 202: // Accepted
           case 203: // Non-Authoritative Information
           case 204: // No content
-            return response.data;
+            if (responseType === "blob") {
```

Comment:
We shouldn't need to add exceptions like this.

Reply:
It's needed because we need the headers for the filename.

task involved.
https://rosebud-iot.atlassian.net/browse/SWEEPR-2924